app/sync.js

```diff
@@ -325,7 +324,18 @@ module.exports.init = function (initialState) {
     if (!records || !records.length) {
       return
     }
-    syncUtil.applySyncRecords(records)
+    let showBookmarksToolbar = false
+    if (!initialState.seed) {
+      // syncing for the first time
```

Comment:
i didn't do that because the bookmarks toolbar could show up noticeably earlier than when any bookmarks are actually created (for instance when syncing a large set of records). but it does make the code simpler.